### DISCUSSION TITLE
should i use uwebsockets to send static assets instead of express?

### DISCUSSION THREAD

**Discussion by ggg7125** [uNetworking/uWebSockets.js#713] [General]
i am very inexperienced. but i see some code in uWebsocket's docs that look like it might be serving static assets. is that something i should be doing? then i can drop express from my project (i think).

i am trying to make a basic game where i will be using uWebsockets for client/server communication and im not sure if i even need anything other than uWebsockets to do so.

**Comment by ghost** [comment]
I know there are a few IO-games made with uWS and yes you certainly can serve static assets with it. I think most of the top IO games are actually made with uWS now that I think about it.
